src/zope/testrunner/options.py

```diff
@@ -296,6 +296,11 @@ def _regex_search(s):
 avoid corrupting the subunit stream.
 """)
 
+reporting.add_argument(
+    '--xml', action='store_true', dest='xmlOutput', help="""\
+If given, XML reports will be written to the current directory.
```

Comment:
Done!

It will be a backwards incompatible change with regards to what `collective.xmltestreport` was doing.

We should mention it on the README, so whoever migrates from `collective.xmltestreport` to `zope.testrunner` with the new `--xml` that they know it is not a boolean, but a path 📚

Reply:
I updated the documentation to mention the new option and added an entry on CHANGES as well.

Reply:
Thank you!